ansible/inventory/env/group_vars/all.yml

```diff
@@ -62,30 +61,20 @@ postgres_address_space: 0.0.0.0/0 # Postgres trust address space
 lp_composite_search: "http://{{ groups['composite-search-cluster'][0] }}:9200" # Composite Cluster ip of LP
 lp_composite_search_host: "{{ groups['composite-search-cluster'][0] }}"
 lp_search: "http://{{private_ingressgateway_ip}}/search"
-lp_url: http://{{ groups['learning'][0] }}:8080/learning-service
 service:
   search:
     url: http://{{private_ingressgateway_ip}}/search
     path: /v3/search
 
-cassandra_hierarchy_store_prefix: "{{env}}_"
```

Comment:
this variable seems to be used in config file and that config is used in Constants.scala